searx/engines/bing.py

```diff
@@ -119,6 +119,8 @@ def response(resp):
                 url_to_resolve.append(url.replace('&ntb=1', '&ntb=F'))
                 url_to_resolve_index.append(i)
                 url = None  # remove the result if the HTTP Bing redirect raise an exception
+        elif url.startswith('/rebates/welcome?'):
+            url = parse_qs(urlparse(url).query)["url"][0]
```

Comment:
> I'm not a Python programmer, so I don't know how to do error-handling etc in Python.

Name `parse_qs`, the function is not imported.  Whenever a URL starts with `/rebates/welcome?` the engine will raise an error.

@AlyoshaVasilieva since you have not seen such error messages in your test scenarios I suppose there was no URL started with `/rebates/welcome?` in your tests --> this PR does not fix https://github.com/searxng/searxng/issues/1764 --> I'm going to close this PR.

Anyway thanks a lot for having an I on this topic, your https://github.com/searxng/searxng/issues/1764#issuecomment-1417990531 helped me to find an other issue :+1: 

- https://github.com/searxng/searxng/pull/2329